sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -950,6 +951,25 @@ class AzureDevopsArtifactsType:
     ARTIFACT = "artifact"
 
 
+class DataIndexTypes:
+    """DataIndexTypes is an enumeration of values for the types out indexes which can be written to by DataIndex."""
+
+    ACS = "acs"
+    """Azure Cognitive Search index type."""
+    PINECONE = "pinecone"
+    """Pinecone index type."""
+    FAISS = "faiss"
+    """Faiss index type."""
+
+
+class IndexInputType(object):
```

Comment:
```suggestion
class IndexInputType:
```

Subclassing object isn't necessary in Python 3

Reply:
thanks, this is corrected now